fhttp/http_client.go

```diff
@@ -135,6 +135,10 @@ type HTTPOptions struct {
 	HTTPReqTimeOut time.Duration // timeout value for http request
 
 	UserCredentials string // user credentials for authorization
+
+	ContentType string // indicates request body type
+	Payload     string // body for http request
```

Comment:
this should probably be `byte[]` and also test that indeed binary gets sent cleanly (and echo'ed back) something like AB\0CD (ie bytes 41 42 00 43 44  don't get truncated at the NUL; another test would be invalid utf8 sequence)